README.md

```diff
@@ -8,7 +8,7 @@ This code is open source software licensed under the [Apache 2.0 License]("http:
 
 ## Endpoints
 
-```POST /ctc/registerncf/v1```
+```POST /ncts5/noticeofcrossingfrontierxi/v1```
```

Comment:
This is changing it from a GB url to an XI url. Looks like the README only lists one endpoint (out of GB and XI). Perhaps we should have both endpoints listed in the README (could perhaps be at the top of the same section, so we don't have to duplicate everything below).

Reply:
Yes, it is raised in https://github.com/hmrc/register-ncf-stub/pull/45

Reply:
that's fine as long as we keep both lines on merge of the second PR